custom_components/ha-glocaltokens/entity.py

```diff
@@ -12,24 +13,11 @@ def __init__(self, coordinator, config_entry):
         super().__init__(coordinator)
         self.config_entry = config_entry
 
-    @property
-    def unique_id(self):
-        """Return a unique ID to use for this entity."""
-        return self._name
-
     @property
     def device_info(self):
         return {
-            "identifiers": {(DOMAIN, self.unique_id)},
-            "name": NAME,
+            "identifiers": {(DOMAIN, self._name)},
+            "name": f"{DEFAULT_NAME} {self._name}",
             "model": VERSION,
             "manufacturer": MANUFACTURER,
         }
-
-    @property
-    def device_state_attributes(self):
-        """Return the state attributes."""
-        return {
-            "id": str(self.unique_id),
-            "integration": DOMAIN,
-        }
```

Comment:
Same goes for this method, it might be required for HA. At least it would make us more consistent with the HA base code. Perhaps as the integration grows we can even make use of it

Reply:
See sensor.py instead. The reason i have moved it is so each sensor created in HA is listed under each device instead of multiple devices for the same device in the device registry. I dont know if there is a better way to implement it

Reply:
That's actually a better design! Let's keep it this way